recipe/build.sh

```diff
@@ -79,6 +79,7 @@ PYTHON_UNSET_INSTALL="-DOPENCV_PYTHON${PY_UNSET_MAJOR}_INSTALL_PATH=${SP_DIR}"
 
 # FFMPEG building requires pkgconfig
 export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$PREFIX/lib/pkgconfig
+export OpenBLAS_HOME=$PREFIX
```

Comment:
This should be removed

Reply:
@isuruf , we are targeting OpenBLAS case here: 

https://github.com/opencv/opencv/blob/b2abd8ca4197744174d34d96e3053f8e0d7fad54/cmake/OpenCVFindLAPACK.cmake#L89-L98

which will call this file:

https://github.com/opencv/opencv/blob/master/cmake/OpenCVFindOpenBLAS.cmake

which uses environment variable `OpenBLAS_HOME` or hardcoded paths for detection.